auth-api/src/auth_api/resources/user.py

```diff
@@ -138,6 +138,32 @@ def get():
         return response, status
 
 
+@cors_preflight('GET,OPTIONS,DELETE')
+@API.route('/<path:username>/otp', methods=['GET', 'OPTIONS', 'DELETE'])
```

Comment:
Is it user name? username have different mapping in different environment (not by design). Some env may have `/` in username. Did you try this whether it works for all cases ?

Reply:
we are using username in other apis as well..most of the user maangement apis by staff/others has this pattern..
the specific / is addressed by the path i guess... i think we discussed this and found the path works..

Reply:
Is the username be unique key in db?

Reply:
Its not the primary key..but i guess it will be unique..